ADR/prettier.md

```diff
@@ -0,0 +1,47 @@
+# Prettier
```

Comment:
Is there any reason to keep this ADR? I see that the upgrade was successful without any implications. If there's nothing to discuss in the future I would say there's no need to have an ADR, or even delete the existing ADR if the decision or technical problem discussed in the document has already been solved.

Reply:
This is actually an error on my side, thanks for catching it. Let me add the AD record. 👍

Reply:
Actually, I think it's important to keep this as a record of when was the last time we updated and if there were any important things to mention. What do you think, should we keep it?